doc/development/submitting/submitting_c.md

```diff
@@ -261,9 +261,11 @@ sent to any of these functions will be printed to stderr.
 
 G_message() output is not expected to be sent to pipe or file.
 
-Always use the gettext macros with _("") for user messages, example:
+Always use the gettext macros with \_("") for user messages, example:
 
 ```c
+#include "glocale.h"
+
 G_fatal_error(_("Vector map <%s> not found"), name);
 ```
 
```

Comment:
GTC: Here the examples are using "GTC" only. The code has a mixture of "GTC" and "GTC:". I see "GTC:" in your new code. Do you want to switch to that here too? "GTC:" seems like a better practice to me.

Reply:
Good catch! I looked into it and it does not matter for comment extraction. Comments are extracted to the POT file as a whole line thus we can freely decide if it should be "GTC" or "GTC:"

Reply:
I changed the text to the "GTC:"